src/assets/scss/components/storiesList.scss

```diff
@@ -31,11 +31,6 @@
     align-items: center;
     display: flex;
   }
-
-  .category-text {
-    color: $hyper-link-color;
-    font-weight: bold;
```

Comment:
I think we should keep it bold atleast? What do you suggest @kailash360 ?

Reply:
@kbhutani0001 I have pushed the changes